pkg/controllers/binding/common.go

```diff
@@ -193,6 +197,32 @@ func RecordAppliedOverrides(cops *overridemanager.AppliedOverrides, ops *overrid
 	return annotations, nil
 }
 
+// mergeConflictResolution determine the conflictResolution annotation of Work: preferentially inherit from RT, then RB
+func mergeConflictResolution(workload *unstructured.Unstructured, conflictResolutionInBinding policyv1alpha1.ConflictResolution,
```

Comment:
Can you help add a ut to cover this method?

Reply:
done